### ISSUE TITLE
Graphiql is broken

### ISSUE THREAD

**Issue by zirho** [sysgears/apollo-universal-starter-kit#284]
![image](https://user-images.githubusercontent.com/1251296/28055591-7f9a9330-65ce-11e7-9b12-e3f80ec28662.png)

```
Uncaught TypeError: Cannot read property 'SubscriptionClient' of undefined
```

I just pulled down the repo and yarn installed.
probably, `subscriptions-transport-ws` has some issues again?

**Comment by vlasenko** [comment]
@zirho I cannot reproduce this with latest version of this starter kit.

**Comment by vlasenko** [comment]
https://github.com/sysgears/apollo-universal-starter-kit/blob/cc4c1fb4d1a00ab702f7c8141d3b302fefeccd10/src/client/app/main.jsx#L60-L63

I guess adding the lines:
``` js
ReactGA.set({ page: window.location.pathname });
ReactGA.pageview(window.location.pathname);
```
below 
``` js
ReactGA.initialize('UA-000000-01');
```
will fix the problem.

Better yet to have a function:
``` js
const logPageView = () => {
  ReactGA.set({ page: window.location.pathname });
  ReactGA.pageview(window.location.pathname);
};
```
and call this function both after `ReactGA.initialize` and inside `history` listener

**Comment by zirho** [comment]
Let me try again with yarn cache clean.

**Comment by zirho** [comment]
![image](https://user-images.githubusercontent.com/1251296/28081458-2880e59e-6624-11e7-8e0a-87ec95a49dce.png)

```
 /Projects/sideProjects/apollo-universal-starter-kit   master  rm -rf node_modules yarn.lock && yarn cache clean && yarn && yarn seed & yarn watch
yarn cache v0.27.5
success Cleared cache.
Done in 8.93s.
yarn install v0.27.5
info No lockfile found.
[1/4] Resolving packages...
warning exp > lodash-node@3.10.2: This package is discontinued. Use lodash@^4.0.0.
warning xdl > auth0-js > xtend > object-keys@0.4.0:
warning react-native > xcode > node-uuid@1.4.7: Use uuid module instead
[2/4] Fetching packages...
███████████████░░░░░░░░░░░░░░░░░░░░░░ 969/15
[3/4] Linking dependencies...
warning "react-dom@16.0.0-alpha.13" has incorrect peer dependency "react@^16.0.0-alpha.13".
[4/4] Building fresh packages...
success Saved lockfile.
Done in 72.07s.
[1] 76441
yarn seed v0.27.5
yarn watch v0.27.5
$ node tools/webpack.entry watch
$ npm run migrate && knex seed:run

> apollo-universal-starter-kit@1.0.0 migrate /Projects/sideProjects/apollo-universal-starter-kit
> knex migrate:latest

Using environment: development
Already up to date
Using environment: development
Hash for vendor_web DLL file ./node_modules/react-transition-group/index.js has changed, need to rebuild it
Generating vendor_web DLL bundle with modules:
["apollo-client","apollo-logger","bootstrap","cors","graphql","graphql-tag","history","immutability-helper","lodash","minilog","persistgraphql","prop-types","react","react-apollo","react-dom","react-ga","react-helmet","react-hot-loader","react-redux","react-router","react-router-dom","react-router-redux","react-transition-group","reactstrap","redux","redux-devtools-extension","redux-form","styled-components","subscriptions-transport-ws"]
Ran 2 seed files
/Projects/sideProjects/apollo-universal-starter-kit/src/server/database/seeds/counter.js
/Projects/sideProjects/apollo-universal-starter-kit/src/server/database/seeds/post.js
Done in 3.71s.
[1]  + 76441 done       yarn seed
webpack-for-web-dll Time: 5876ms
[HPM] Proxy created: !/*.hot-update.{json,js}  ->  http://192.168.20.161:8080
webpack-for-web-frontend Webpack web-frontend dev server listening on http://192.168.20.161:3000
webpack-for-web-frontend debug Webpack dev server is waiting for backend to start...
webpack-for-backend Starting backend
webpack-for-backend Time: 11901ms

/Projects/sideProjects/apollo-universal-starter-kit/node_modules/react-apollo/node_modules/react-dom/cjs/react-dom-server.development.js:3534
  ReactCurrentOwner: ReactInternals.ReactCurrentOwner
                                   ^
TypeError: Cannot read property 'ReactCurrentOwner' of undefined
    at Object.<anonymous> (/Projects/sideProjects/apollo-universal-starter-kit/node_modules/react-apollo/node_modules/react-dom/cjs/react-dom-server.development.js:3534:36)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Projects/sideProjects/apollo-universal-starter-kit/node_modules/react-apollo/node_modules/react-dom/server.js:6:20)
    at Module._compile (module.js:570:32)
webpack-for-backend Backend has been stopped
```

![image](https://user-images.githubusercontent.com/1251296/28081509-5934dd58-6624-11e7-9dd6-848c772d9ab3.png)

Am I doing something wrong?
Let me know what I can do more. 
Thank you

**Comment by vlasenko** [comment]
@zirho Yes, looks like your `package.json` is modified, it uses `react-dom@16.0.0-alpha.13`

**Comment by zirho** [comment]
That's kinda weird since I can't find `16.0.0` anywhere in package.json and yarn.lock at all.
Let me do some research. Thanks!

**Comment by vlasenko** [comment]
@zirho What `yarn list` tells ?

**Comment by zirho** [comment]
![image](https://user-images.githubusercontent.com/1251296/28082361-2215f2d2-6627-11e7-92dd-5c470c479e7d.png)

I pulled down fresh repo and did it again and it worked. 

And I went and open graphiql again and got error again.
I tried it in chrome incognito because now I knew it works fine for others.

I found that my ad blocker blocks some js files. 
Once I disable the blocker it started working fine. 

Sorry about the mess and thanks for the help!
